Hardware/Hardware/CardReader/StripeCardReader/PaymentIntent+Stripe.swift

```diff
@@ -33,5 +33,10 @@ protocol StripePaymentIntent {
 }
 
 
-extension StripeTerminal.PaymentIntent: StripePaymentIntent { }
+extension StripeTerminal.PaymentIntent: StripePaymentIntent {
+    // TODO: Temporarily added here for conformance
+    var stripeId: String {
+        ""
+    }
```

Comment:
> SCPPaymentIntent.stripeId is now nullable to support offline payments.

Making stripeID nullable involves more extensive changes that we can skip for the moment, since we do not support offline payments and will be non-null in all cases.